dataland-backend/src/main/kotlin/org/dataland/datalandbackend/configurations/ComplexExampleSupporter.kt

```diff
@@ -1,31 +0,0 @@
-package org.dataland.datalandbackend.configurations
-
-import io.swagger.v3.oas.models.OpenAPI
-import io.swagger.v3.oas.models.media.Schema
-import org.springdoc.core.customizers.OpenApiCustomizer
-import org.springframework.stereotype.Component
-
-/**
- * This class adds additional properties (ref) to fields of a more complex type (e.g. Map<...,...>) that are equipped
- * with an example, making it in some cases not possible to keep the information about the corresponding type used in
- * the map
- */
-
```

Comment:
This one is no longer necessary because the structure of the field was changed, correct?

Reply:
Before, a sector was mapped to an ExtendedDataPoint<BigDecimal>. This caused the problem that the custom example added to the applicableHighImpactClimateSectors field in the backend was added to all ExtendedDataPoint<BigDecimal> and not only to the single desired one. As we now have a custom object which occurs only once, this additional support here became redundant.

Reply:
Ok